plugin/src/main/java/org/bonitasoft/plugin/analyze/ConnectorResolver.java

```diff
@@ -47,27 +59,21 @@
 import org.xml.sax.SAXException;
 
 import com.google.inject.Singleton;
-import com.strobel.assembler.metadata.DeobfuscationUtilities;
-import com.strobel.assembler.metadata.ITypeLoader;
-import com.strobel.assembler.metadata.JarTypeLoader;
-import com.strobel.assembler.metadata.MetadataSystem;
-import com.strobel.assembler.metadata.TypeDefinition;
-import com.strobel.assembler.metadata.TypeReference;
-import com.strobel.decompiler.DecompilerSettings;
 
 @Named
 @Singleton
 public class ConnectorResolver {
 
-    public static final String CONNECTOR_TYPE = "org/bonitasoft/engine/connector/Connector";
-    public static final String FILTER_TYPE = "org/bonitasoft/engine/filter/UserFilter";
-    public static final String ABSTRACT_CONNECTOR_TYPE = "org/bonitasoft/engine/connector/AbstractConnector";
-    public static final String ABSTRACT_FILTER_TYPE = "org/bonitasoft/engine/filter/AbstractUserFilter";
+    private static final String CONNECTOR_TYPE = "org.bonitasoft.engine.connector.Connector";
+    private static final String FILTER_TYPE = "org.bonitasoft.engine.filter.UserFilter";
+    private static final String ABSTRACT_CONNECTOR_TYPE = "org.bonitasoft.engine.connector.AbstractConnector";
+    private static final String ABSTRACT_FILTER_TYPE = "org.bonitasoft.engine.filter.AbstractUserFilter";
 
     private static final Set<String> FILTER_TYPES = Set.of(FILTER_TYPE, ABSTRACT_FILTER_TYPE);
     private static final Set<String> CONNECTOR_TYPES = Set.of(CONNECTOR_TYPE, ABSTRACT_CONNECTOR_TYPE);
```

Comment:
Shouldn't you also add REST API extension to this list in orger to manage all extension points ?

Reply:
Nope